src/soilId/soilIdSlice.ts

```diff
@@ -17,44 +17,22 @@
 
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { setUsers } from 'terraso-client-shared/account/accountSlice';
-import type {
-  DepthDependentSoilDataNode,
-  DepthInterval,
-  ProjectDepthIntervalNode,
-  ProjectSoilSettingsNode,
-  SoilDataDepthIntervalNode,
-  SoilDataNode,
-} from 'terraso-client-shared/graphqlSchema/graphql';
 import { setProjects } from 'terraso-client-shared/project/projectSlice';
 import { setSites } from 'terraso-client-shared/site/siteSlice';
 import * as soilIdService from 'terraso-client-shared/soilId/soilIdService';
+import {
+  CollectionMethod,
+  DepthInterval,
+  ProjectSoilSettings,
+  SoilData,
+  SoilPitMethod,
+} from 'terraso-client-shared/soilId/soilIdTypes';
 import {
   createAsyncThunk,
   dispatchByKeys,
 } from 'terraso-client-shared/store/utils';
 
-export const soilPitMethods = [
-  'soilTexture',
-  'soilColor',
-  'carbonates',
-  'ph',
-  'soilOrganicCarbonMatter',
-  'electricalConductivity',
-  'sodiumAdsorptionRatio',
-  'soilStructure',
-] as const;
-export const collectionMethods = [
-  'slope',
-  'verticalCracking',
-  ...soilPitMethods,
-  'landUseLandCover',
-  'soilLimitations',
-  'photos',
-  'notes',
-] as const;
-
-export type SoilPitMethod = (typeof soilPitMethods)[number];
-export type CollectionMethod = (typeof collectionMethods)[number];
+export * from 'terraso-client-shared/soilId/soilIdTypes';
```

Comment:
question do we want to export * or should we enumerate the exports?

Reply:
we want to export `*`. this code re-org was a case of semantically everything belonging in `soilIdSlice` but the file was just getting too unwieldy so i split it off and re-exported everything. ideally i would also make it so downstream clients can't import from `soilIdTypes` so there's a canonical import for dependents but i'm not sure how to do that and there's more important things to do rn